cmd/api/src/queries/graph.go

```diff
@@ -132,7 +132,7 @@ func BuildEntityQueryParams(request *http.Request, queryName string, pathDelegat
 }
 
 type Graph interface {
-	GetAssetGroupComboNode(ctx context.Context, owningObjectID string, assetGroupTag string) (map[string]any, error)
+	GetAssetGroupComboNode(ctx context.Context, owningObjectID string, assetGroupTag string, etacEnabled bool, user model.User) (map[string]any, error)
```

Comment:
Let's make the interface instead like: 
```suggestion
	GetAssetGroupComboNode(ctx context.Context, owningObjectID string, assetGroupTag string, environmentsFilter []string) (map[string]any, error)
```
This prevents bleed of ETAC into our interface and reduces arguments. Additionally, nil or a nil slice can easily be passed in if there is no need for environments filtering